### PR TITLE
Closes #358 | Create dataset loader for GKLMIP Product Sentiment

### LINKED ISSUE #358: Create dataset loader for GKLMIP Product Sentiment
Dataloader name: `gklmip_sentiment/gklmip_sentiment.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?gklmip_sentiment

|     Dataset| gklmip_sentiment |
|-------------|---|
| Description | The GKLMIP Product Sentiment Dataset is a Burmese dataset for sentiment analysis. It was created by crawling comments on an e-commerce website. The sentiment labels range from 1 to 5, with 1 and 2 being negative, 3 and 4 being neutral, and 5 being positive. |
| Subsets | - |
| Languages | mya |
| Tasks | Sentiment Analysis |
| License | Unknown (unknown) |
| Homepage | https://github.com/GKLMIP/Pretrained-Models-For-Myanmar/tree/main |
| HF URL | - |
| Paper URL | https://link.springer.com/chapter/10.1007/978-3-030-92310-5_52 |

### PR DESCRIPTION
**Pull request by djanibekov**
Closes #358 

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

**Review by MJonibek — APPROVED** [review]
Checked, LGTM

File: seacrowd/sea_datasets/gklmip_sentiment/gklmip_sentiment.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+
+        urls = _URLS[_DATASETNAME]
+        data_dir = dl_manager.download_and_extract(urls)
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                # Whatever you put in gen_kwargs will be passed to _generate_examples
```
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/gklmip_sentiment/gklmip_sentiment.py:116]:
```suggestion
```

File: seacrowd/sea_datasets/gklmip_sentiment/gklmip_sentiment.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir, "product_sentiment_dataset_train.json"),
+                    "split": "train",
```
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/gklmip_sentiment/gklmip_sentiment.py:119]: You don't need to include the `split` keyword if you won't use it in the `generate_examples` function

**Review by ljvmiranda921 — APPROVED** [review]
Data loader works as expected, just have few small nits :)

**Comment by ljvmiranda921** [comment]
Hi @djanibekov ! Do you need any help with the changes? The data loader looks good in general, just have some small nits before we merge. :)

**Comment by djanibekov** [comment]
@ljvmiranda921 I'm sorry for long response, lost your message in my email inbox

**Review by ljvmiranda921 — APPROVED** [review]
